src/screens/Election/ElectionApplication.js

```diff
@@ -42,201 +32,170 @@ class ElectionApplication extends Component {
+				<Text style = { [fontLarge, textColor] }>
+					{ firstName } { lastName }
+				</Text>
+				<Text style = { [fontLarge, textColor] }>Plan:</Text>
+				{ this.renderError() }
+				<Input
+					numberOfLines = { 10 }
+					style = { halfFlex }
+					textAlignVertical = "top"
+					maxLength = { 250 }
+					blurOnSubmit = { true }
+					multiline = { true }
+					placeholder = "Please write your plan for members to read."
+					value = { candidatePlan }
+					onChangeText = { (plan) => this.setState({ candidatePlan: plan }) }
+				/>
 			</View>
 		);
 	}
 
 	renderError() {
 		if (this.props.error)
 			return (
-				<View>
-					<Text style = { styles.errorTextStyle }>
-						{ this.props.error }
-					</Text>
-				</View>
+				<Text style = { styles.errorfontLarge }>
+					{ this.props.error }
+				</Text>
 			);
 	}
 
 	_keyExtractor = (item, index) => index;
 
 	renderPositions(positionsArray) {
-		if (!this.state.isListShow) return null;
-
 		return (
-			<View>
-				<FlatList
-					data = { positionsArray }
-					extraData = { this.state }
-					keyExtractor = { this._keyExtractor }
-					renderItem = { ({ item }) => {
-						if (this.props.applied)
-							return this.renderPositionApplied(item);
-						else
-							return this.renderPositionComponent(item);
-					} }
-				/>
-			</View>
+			<FlatList
+				data = { positionsArray }
+				extraData = { this.state }
+				keyExtractor = { this._keyExtractor }
+				renderItem = { ({ item }) => this.renderPositionComponent(item) }
+			/>
 		);
 	}
 
 	renderPositionComponent(item) {
 		const {
-			button,
-			textStyle,
+			fontLarge,
+			fontSmall,
 			textColor,
+			fullFlex,
 			positionContainer
 		} = styles;
-		this.state.application = "Submit";
 
 		return (
-			<View style = { positionContainer }>
-				<View style = {{ marginBottom: 10 }}>
-					<Text style = { [textStyle, textColor] }>Position: { item.title }</Text>
-				</View>
-				<View style = {{ marginLeft: 12, marginRight: 10, marginBottom: 8 }}>
-					<Text style = { [textStyle, textColor] }>Role: { item.description }</Text>
-				</View>
-				<View style = { button }>
-					<Button
-						title = { `Apply for ${item.title}` }
-						onPress = { () => {
-							this.setState({ isListShow: false, isApplyShow: true, applyPos: item.title });
-						} }
-						 />
-				</View>
-			</View>
-		);
-	}
-
-	renderPositionApplied(item) {
-		const {
-			textStyle,
-			textColor,
-			positionContainer
-		} = styles;
-
-		return (
-			<View style = { positionContainer }>
-				<View style = {{ marginBottom: 10 }}>
-					<Text style = { [textStyle, textColor] }>Position: { `${item.title}` }</Text>
-				</View>
-				<View style = {{ marginLeft: 12, marginRight: 10 }}>
-					<Text style = { [textStyle, textColor] }>Role: { `${item.description}` }</Text>
+			<TouchableOpacity
+				onPress = { () => {
+					this.setState({ currentlyApplying: true, positionSelected: item.title });
+				} }
+				style = { [positionContainer, fullFlex] }>
```

Comment:
Consider leaving the ending tag `>` in a newline to differentiate between the inner blocks.